pyiron/atomistics/structure/atoms.py

```diff
@@ -1997,20 +1995,22 @@ def get_distances_array(self, a0=None, a1=None, mic=True, vector=False):
         """
         if a0 is None and a1 is not None:
             a0 = a1
-            a1 = None
         if a0 is None:
-            a0 = self.positions
+            a0 = a1 = self.positions
         a0 = np.array(a0).reshape(-1, 3)
-        if a1 is not None:
-            a1 = np.array(a1).reshape(-1, 3)
+        a1 = np.array(a1).reshape(-1, 3)
+        x = np.einsum(
```

Comment:
A nitpick on coding style: It'll be nice to have a better descriptive variable than `x`. I know we can't change `a0` and `a1` since we have to be compatible with ASE but maybe something like `diff_relative` would make this much more readable

Reply:
Actually `get_distances` in [ase](https://wiki.fysik.dtu.dk/ase/_modules/ase/geometry/geometry.html#get_distances) is `p0` and `p1`, so we are anyway somewhat changing the names...

Reply:
In that case we should also refractor to use `p0` and `p1`. This will help make our the transition from ase workflows to pyiron seamless

Reply:
Done!

Reply:
Oh no it's `p1` and `p2`...

Reply:
Done!